src/content/ja/2021/performance.md

```diff
@@ -0,0 +1,509 @@
+
+{{ figure_markup(
+  image="performance-TTFB-by-rank.png",
+  caption="TTFBのランク別パフォーマンス",
+  description="各ランキンググループのTTFBパフォーマンスを示す積み上げ棒グラフ。上位1,000サイトでは、32%が「良い」、63%が「改善必要」、5%が「悪い」となっています。上位10,000サイトでは、27%が「良い」、66%が「要改善」、7%が「悪い」でした。上位10万サイトでは、22%が「良い」、65%が「改善必要」、13%が「悪い」でした。100万人規模では、「良い」が18％、「改善必要」が62％、「悪い」が20％。全体では、「良い」18％、「改善が必要」58％、「悪い」24％であった。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=829561432&format=interactive",
+  sheets_gid="1988323604",
+  sql_file="web_vitals_by_rank.sql"
+  )
+}}
+
+ランクについては、高ランクのサイトほどTTFBが速かった。その理由のひとつは、これらの多くは、パフォーマンスを優先するためのより多くのリソースを持つ大企業であることでしょう。これらの企業は、サーバーサイドのパフォーマンスの向上や、エッジCDNを通じたアセット配信に注力している可能性があります。もうひとつの理由は、選択バイアスです。上位のオリジンは、サーバーが近い地域、つまり低遅延の地域でより多くアクセスされる可能性があります。
+
+もう1つ可能性があるとすれば、CMSの導入が関係している。[CMS編](./cms)では、ランク別のCMS導入状況を示しています。
+
+{{ figure_markup(
+  image="cms-adoption-by-rank.png",
+  caption="ランク別CMS採用状況",
+  description="ランキングのカテゴリーごとに、CMSを利用しているウェブサイトの割合を棒グラフで示したもの。モバイルでは、1,000位以内が7%、10,000位以内15%、10万位以内20%、100万位以内29%、全サイトでは42%となっています。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRzYkQYaBTpD5FKloDXsdb32Y2pjdxFZq_LoekECgTy53F-dRATm9wFFJ3dDVDS8_cGn0h2mKDBjgdM/pubchart?oid=409766380&format=interactive",
+  sheets_gid="https://docs.google.com/spreadsheets/d/1gAJh4VcSEU6-6aQxBiTFl-7cbyHukfdhg-Iaq9y5pnc/#gid=158167539",
+  sql_file="../cms/cms_adoption_by_rank.sql"
+  )
+}}
+
+「すべて」のグループでは42%のページ（モバイル）がCMSを使用しているのに対し、上位1,000サイトでは7%の導入にとどまっています。
+
+次に、上位5つのCMSを順位別に見ると、WordPressが「全ページ」の33.6%を占め、もっとも導入が進んでいることがわかります。
+
+{{ figure_markup(
+  image="top-cmss-by-rank.png",
+  caption="CMSのランキングトップ5",
+  description="各ランキングカテゴリーにおける上位CMSのシェア率を棒グラフで示したもの。WordPressは、上位1,000位で3.1%、1万位では8.6%、10万位で13.2%、100万位では21.1%、そして全サイトの33.6%を占めています。残りのCMSは全ランキングに占める割合が3%未満でした。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRzYkQYaBTpD5FKloDXsdb32Y2pjdxFZq_LoekECgTy53F-dRATm9wFFJ3dDVDS8_cGn0h2mKDBjgdM/pubchart?oid=1745757207&format=interactive",
+  sheets_gid="https://docs.google.com/spreadsheets/d/1gAJh4VcSEU6-6aQxBiTFl-7cbyHukfdhg-Iaq9y5pnc/#gid=670045665",
+  sql_file="../cms/top_cms_by_rank.sql"
+  )
+}}
+
+最後に、<a hreflang="en" href="https://datastudio.google.com/s/o6zLzlTpWaI">Core Web Vitalsテクノロジーレポート</a> を見ると、各CMSが指標別にどのようなパフォーマンスを見せてくれるかがわかります。
+
+{{ figure_markup(
+  link="https://datastudio.google.com/s/o6zLzlTpWaI",
+  image="cms_ttfb_cwv_report.png",
+  caption='CMSによる良いTTFBを持つオリジン (<a hreflang="en" href="https://datastudio.google.com/s/o6zLzlTpWaI">Core Web Vitalsテクノロジーレポート</a>)',
+  description="WordPress、Wix、Squarespaceの「Origin Having Good TTFB」の時系列チャート。7月は、WordPressとWixが5％、Squarespaceが20％でした。"
```

Comment:
Should "Origin Having Good TTFB" be translated?

Reply:
Thanks, I translated.